packages/trader/src/Stores/Modules/Trading/Helpers/proposal.ts

```diff
@@ -39,7 +39,9 @@ type TObjExpiry = {
 type TValidationParams =
     | {
           validation_params?: {
-              max_payout?: string;
+              payout?: {
```

Comment:
For Acc we are still getting `max_payout` in proposal
<img width="1674" alt="Screenshot 2024-09-17 at 10 28 27 AM" src="https://github.com/user-attachments/assets/f58fa470-248b-46bc-bab0-f33f92ebea8b">
<img width="1674" alt="Screenshot 2024-09-17 at 10 31 15 AM" src="https://github.com/user-attachments/assets/f35e5755-7cf3-4c27-801f-7337bdf30c2b">

Reply:
Ah, okey, seems that we are not suppose to show max_payout for Acc

Reply:
ok will keep. Will have to extract out max_payout separately if we want to show max_payout in the future. for now payout: { max } works only